apply-notch-hax.sh

```diff
@@ -36,19 +54,20 @@ for file in $(grep '^diff' $DIFF | grep -Eo '\ b/.*' | cut -c 4-); do
 	cp /$file $WORK/root/$file
 done
 
-cd root/
+pushd root/
```

Comment:
Any reason for these `pushd` and `popd`'s other than being more verbose? Imho this just clutters the script output

Reply:
Mostly personal preference, and avoiding hardcoded `cd ..`'s which could break if root isn't *exactly* one folder deep. Could add `> /dev/null`, but probably easiest to just revert back for something like this.